.github/workflows/reusable_basic.yml

```diff
@@ -124,7 +135,19 @@ jobs:
     - name: Install apt packages
       run: |
         sudo apt-get update
-        sudo apt-get install -y clang cmake libnuma-dev lcov
+        sudo apt-get install -y clang libnuma-dev lcov
+
+    - name: Install cmake (latest)
+      if: matrix.cmake_ver == 'latest'
+      run: |
+        sudo apt-get install -y cmake
```

Comment:
I think this is useless, because the CMake is installed by default. In case of Ubuntu 22.04 the default version is CMake 3.31 in GitHub Images:
https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md?plain=1